OptimoveSDK/Sources/Classes/PreferenceCenter/Models/PreferenceUpdateRequest.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+public struct PreferenceUpdateRequest: Codable {
```

Comment:
can't we update Android for this one? `PreferenceUpdate` on its own doesn't sound super clear. It took Eli a few extra seconds to understand what it is, which is a sign that it could use an improvement :p

Reply:
well argued 😸  I will leave it as PreferenceUpdate, and if we really don't like it in the long run, I'll consider myself responsible for updating all sdks